internal/controller/model_nodeport.go

```diff
@@ -23,6 +27,14 @@ import (
 	"github.com/cloudandheat/ch-k8s-lbaas/internal/openstack"
 )
 
+var (
+	ErrUnknownAddressFamily = errors.New("cannot determine the correct IP address family")
+)
+
+var (
+	ErrInvalidIpAddress = errors.New("the string is not a valid textual representation of an IP address")
+)
+
```

Comment:
You can use a single var block for those.

Reply:
Thanks, changed.